openlibrary/plugins/openlibrary/js/patner_ol_lib.js

```diff
@@ -2,20 +2,19 @@ const getIsbnToElementMap = container => {
     const reISBN = /((978)?[0-9][0-9]{10}[0-9xX])|((978)?[0-9]{9}[0-9Xx])/;
     const elements = document.getElementsByClassName(container);
     let isbnElementMap = {};
-    for (let i = 0; i < elements.length; i++) {
-        let e = elements.item(i);
-        let match = e.innerHTML.match(reISBN);
-        if (match) {
-            isbnElementMap[match[0]] = e;
+    Array.from(elements).map((e) => {
```

Comment:
I would do with the syntax for element of elements .. but this also works

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of